src/Jackett.Common/Definitions/hdonly.yml

```diff
@@ -54,6 +54,22 @@ settings:
     type: checkbox
     label: Include DV/HDR10 in filename when a release has multiple HDR formats.
     default: false
+  - name: add_vff_to_title
```

Comment:
Odds are good that if a user wants to see the v* flags in their titles, they would choose to enable all of them.
I suggest that only one checkbox is needed for all 4 flags to be allowed when present to titles
- name:  add_version_francophone
  type: checkbox
  label: Append Version Francophone flags to titles if available (VFF, VFQ, VFI, VF)

Reply:
^ yea, that looks good to me